tools/metricsdocs/metricsdocs_suite_test.go

```diff
@@ -0,0 +1,16 @@
+package main
```

Comment:
should have its own package
see please other *suite_test.go files

Reply:
in the tests of other tool packages, it is the same package, https://github.com/kubevirt/cluster-network-addons-operator/tree/main/tools/bumper

Reply:
ok because it is a standalone tool right

Reply:
yes